### DISCUSSION TITLE
Broken link in 5.2-AlexaHandsfree.md

### DISCUSSION THREAD

**Discussion by abulatkin** [aws-samples/retail-demo-store#285] [General]
I'm going through 5.2-AlexaHandsfree.md and this link [src/alexa/skill-package/custom/en-US.json](../../src/alexa/skill-package/custom/en-US.json) is giving me 404. 
I believe the link should be src/alexa/skill-package/interactionModels/custom/en-US.json

**Comment by anjalisinghanjalisingh689-cloud** [comment]
Retail-demo-store offers digital marketing solutions such as SEO, social media marketing, Google Ads, and content creation. These services help retail businesses increase online visibility, attract customers, and improve sales effectively
Visit Site-https://growwithanjali.info/
